### DISCUSSION TITLE
Some meetings in TSML not showing up in Meeting Guide

### DISCUSSION THREAD

**Discussion by Spsrky** [code4recovery/12-step-meeting-list#1000] [Q&A] [answered]
I'm hoping you can shed some light on a recent problem with meetings not being picked up from TSML into Meeting Guide. A few weeks ago, a nearby District (14) just outside our coverage area (Portland, OR)  asked to be included in our listings  so their members could find meetings on their phones. They sent their list of meetings in a CSV they had carefully formatted so it appeared to conform to the data requirements for TSML uploads. It looked fine to me.

I uploaded the file and the meetings were added as expects - visible in the TSML backend listing and on the website but they never were picked up in Meeting Guide. I tried some simple changes to the location information, thinking the geolocation was thrown off, but no help. I deleted the meetings from District 14 but now we're finding that about 500 of our total 1100 meetings in TSML are now gone from Meeting Guide. 

It seems (?) as though the District 14 upload introduced some formatting glitch into the database that has affected other meetings as well. I have attached copies of the District 14 CSV, an older TSML CSV backup from last July, and a TSML backup from today, since the problem started. I don't know what to look for at this point, hoping you might. Thanks!

Theron
PDXAA.org

[District 14-meeting data dist 14 11-10-22.csv](https://github.com/co
[TSML backup-meetings-2022-12-13.csv](https://github.com/code4recovery/12-step-meeting-list/files/10222192/TSML.backup-meetings-2022-12-13.csv)
de4recovery/12-step-meeting-list/files/10222187/District.14-meeting.data.dist.14.11-10-22.csv)
[TSML backup-meetings-2022-07-22.csv](https://github.com/code4recovery/12-step-meeting-list/files/10222196/TSML.backup-meetings-2022-07-22.csv)

**Comment by joshreisner** [comment, marked as answer]
Hi @Spsrky have you reached out to [appsupport@aa.org](appsupport@aa.org)? They can provide you with a report of which meetings are not being listed and why. We don't have access to that information.

Most likely the top reason will be conflicts with another office. Also you may be listing meetings outside the coverage boundary that they have set for you. Finally, keep in mind that you only ever see 500 meetings a time using the app. You will need to search in a new location (not filter) to see meetings that are further away.

**Comment by Spsrky** [comment]
Thanks, Josh! I've contacted AppSupport, I'll let you know what the outcome is.

In fellowship and service,

Theron
